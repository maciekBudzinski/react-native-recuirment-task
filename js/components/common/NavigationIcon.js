import React from 'react';
import { Icon } from 'native-base';
import PropTypes from 'prop-types';

export const NavigationIcon = ({ iconName }) => <Icon name={iconName} />;

NavigationIcon.propTypes = {
  iconName: PropTypes.string.isRequired,
};
