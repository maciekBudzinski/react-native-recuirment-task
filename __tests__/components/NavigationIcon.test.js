import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationIcon from '../../js/components/common/NavigationIcon';

test('NavigationItem test', () => {
  const props = { iconName: 'add' };
  const snapshot = renderer.create(<NavigationIcon {...props} />).toJSON;
  expect(snapshot).toMatchSnapshot();
});
