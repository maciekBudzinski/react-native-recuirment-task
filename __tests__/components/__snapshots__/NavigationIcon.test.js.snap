// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem test 1`] = `[Function]`;
